sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/administration/ServiceBusAdministrationAsyncClientIntegrationTest.java

```diff
@@ -1025,6 +1051,9 @@ private ServiceBusAdministrationAsyncClient createClient(HttpClient httpClient)
                 .addPolicy(interceptorManager.getRecordPolicy());
         }
 
+        interceptorManager.addSanitizers(TEST_PROXY_SANITIZERS);
+        interceptorManager.addMatchers(TEST_PROXY_REQUEST_MATCHERS);
```

Comment:
Matchers only need to be added in playback mode. As the "Match" only when playing it back.